flash/data/process.py

```diff
@@ -168,7 +313,7 @@ def add_callbacks(self, callbacks: List['FlashCallback']):
         self._callbacks.extend(_callbacks)
 
     @classmethod
-    def load_data(cls, data: Any, dataset: Optional[Any] = None) -> Any:
+    def load_data(cls, data: Any, dataset: Optional[Any] = None) -> Mapping:
         """Loads entire data from Dataset"""
```

Comment:
Add more details. What kind of input can you pass in?

How to create the mapping? 
+code example

Reply:
Done !